dbt_subprojects/daily_spellbook/models/_sector/labels/addresses/bridge/identifiers/labels_bridges_near.sql

```diff
@@ -0,0 +1,19 @@
+{{config(
+        materialized = 'view',
+        alias = 'labels_bridges_near',
```

Comment:
```suggestion
        schema = 'labels',
        alias = 'bridges_near',
```

Reply:
the legacy labels pipeline (the one you're using here in daily subproject) has a default schema assignment in the global config file
https://github.com/duneanalytics/spellbook/blob/main/dbt_subprojects/daily_spellbook/dbt_project.yml#L109-L113

we should avoid using `labels_` again in the table name, since schema already uses it.
also, while schema will automatically be picked up, we've been using best practice to call it out in model directly to make it easier to follow naming standards for anyone verifying code.